tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "include": [".storybook/*", "**/*"],
   "exclude": ["node_modules"],
   "compilerOptions": {
+    "types": ["vite/client"],
```

Comment:
Vite provides useful types defaults, e.g. in this file they have

```
declare module '*.svg' {
  const src: string
  export default src
}
```

which is needed for this PR to work